home/index.js

```diff
@@ -1,39 +1,41 @@
-class Mentee {
+
+// Members class to identify mentees and mentors properties
+class Members {
```

Comment:
Member model contains the following properties:
- title 
- description
- country
- city
- start_hour
- end_hour
- tags (string array)
- social_links (array map each item contains (type (string), value(string)) 
- user_email